content/20.treatments.md

```diff
@@ -505,6 +504,39 @@ Supplementation of vitamin D and maintaining a healthy diet for optimum vitamin
+Therefore, an effective prophylactic therapy for ventilator-associated pneumonia in severe COVID-19 patients would be of significant therapeutic value.
+
+Probiotics are generally synonymous with the treatment of gastrointestinal issues due to their supposed anti-inflammatory and immunomodulatory effects [@doi:10.1016/S2352-4642(19)30182-8]. 
+Notably, gastrointestinal symptoms commonly occur in COVID-19 patients [@doi:10.1053/j.gastro.2020.03.020] and the ACE2 receptor is highly expressed in enterocytes of the ileum and colon indicating that the these organs may be a potential route of infection [@doi: 10.1101/2020.01.30.927806; @doi:10.1038/cr.2016.152]. 
+Indeed, SARS-CoV-2 viral RNA has been detected in human faeces  [@doi:10.1016/S2468-1253(20)30083-2] and faecal-oral transmission of the virus has not yet been ruled out [@doi:10.1016/S2468-1253(20)30048-0]. 
+Rectal swabs of some SARS-CoV-2 positive paediatric patients persistently tested positive for several days despite their nasopharyngeal sample testing negative indicating the potential for faecal viral shedding [@doi:10.1038/s41591-020-0817-4].
+However, there is conflicting evidence for the therapeutic value of various probiotics against the incidence or severity of gastrointestinal symptoms in viral or bacterial infections such as gastroenteritis [@doi:10.1007/s00203-017-1400-3; @doi:10.1056/NEJMoa1802597]. 
+Nevertheless, it has been proposed that the administration of probiotics to COVID-19 patients and healthcare workers may prevent or ameliorate the gastrointestinal symptoms of COVID-19 for which several clinical trials are now preparing to investigate [@clinicaltrials:NCT04420676; @clinicaltrials:NCT04390477; @clinicaltrials:NCT04366180]. 
+
+**Summary.**
+Generally, the efficacy of probiotic use is a controversial topic among scientists. In Europe, EFSA has banned the term probiotics on products labels, which has elicited either criticism for EFSA or support for probiotics among researchers in the field [@doi:10.1016/j.bpg.2015.12.001; @doi:10.1017/S000711451100287X; @doi:10.1038/nrgastro.2014.66]. 
+This is due to the hyperbolical claims placed on the labels of various probiotic products, which lack rigorous scientific data to support their efficacy. 
+Overall, the data supporting probiotics in the treatment or prevention of many different disorders and diseases is not conclusive as the quality of the evidence is generally considered low [@doi:10.1007/s10096-014-2086-y]. However, in the case of probiotics and respiratory infections, the evidence seems to be supportive of their potential therapeutic value.
+Consequently, several investigations are underway to investigate the prophylactic and therapeutic potential of probiotics for COVID-19.
+However, blind use of conventional probiotics for COVID-19 is cautioned against until the pathogenesis of SARS-CoV-2 is further established [@doi:10.1016/S2468-1253(20)30122-9].  
```

Comment:
```suggestion
However, blind use of conventional probiotics for COVID-19 is cautioned against until the pathogenesis of SARS-CoV-2 is further established [@doi:10/d2qq]. 
```

Reply:
Hey @rando2 the suggested doi does not link to the article, does that matter?

Reply:
> Hey @rando2 the suggested doi does not link to the article, does that matter?

Hmm it's working for me, does this link work? <https://doi.org/10/d2qq>

Reply:
> > Hey @rando2 the suggested doi does not link to the article, does that matter?
> 
> Hmm it's working for me, does this link work for you? https://doi.org/10/d2qq

Ah, spot on, thank you. Good to know more about manubot :)